src/app/src/util/util.js

```diff
@@ -185,6 +185,22 @@ export const createPaginationOptionsFromQueryString = (qs) => {
     });
 };
 
+export const createParamsFromQueryString = (qs) => {
```

Comment:
Consider adding a test for this new function.

Reply:
Added tests in fixup b1e998a